docs/migrate.md

```diff
@@ -37,6 +39,8 @@ restart the necessary containers.
 New installations do not need this `kustomization` and existing installations can operate from base again after the
 migration.
 
+### New installations with restrictive security policies
```

Comment:
Is this section actually a migration? Or should this just be part of our Kubernetes documentation for this specific case? 

```suggestion
### New installations: accommodate clusters with restrictive security policies
```